doc/website/getting-started/examples/waitset.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Wait until one or more subscriber has received data
```

Comment:
Getting notified by events like the receiving of data (with the WaitSet).

Reply:
Waiting for data events

Reply:
Active is easier to grasp than passive senteces